common/prompts/skill_selector.json

```diff
@@ -0,0 +1,3 @@
+{
+    "prompt": "Select N_SENTENCES_TO_RETURN the most relevant to the dialog context skills based on the given short descriptions of abilities of different skills of the assistant. Return only names of the selected skills divided by comma.\n\n"
```

Comment:
Select N_SENTENCES_TO_RETURN skills most relevant to the dialog context based on the given short descriptions of abilities of different skills of the assistant. Return only the names of the selected skills divided by comma.\n\n